src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -136,8 +136,17 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
 
   onDisable = (integration: Integration) => {
     let url: string;
-    const [domainName, orgName] = integration.domainName.split('/');
 
+    if (integration.provider.key === 'vercel') {
+      // kind of a hack since this isn't what the url was stored for
+      // but it's exactly what we need and contains the configuration id
+      // e.g. https://vercel.com/dashboard/integrations/icfg_ySlF4UDnHcIPrAAXjGEiwtxo
+      url = integration.configOrganization[0].nextButton.url;
```

Comment:
I don't really like this but it does have the config id. The url works for Team's too but you can also use `https://vercel.com/dashboard/meredithteam/integrations/<config_id>` so I'm not sure if there is going to be a difference.